### ISSUE TITLE
Suggestion - public drinking prohibition zones enforced with crime points and tickets.

### ISSUE THREAD

**Issue by Nextej** [loiste-interactive/Obenseuer-Issues#1019]
Make some areas in the world prohibited from drinking alcohol, as in "in public", consumption would meet with an added up crime points of an associated category and bills/tickets or drinking delivered by the mail if possible. Some examples of such areas could be Kolhola Hostel and public stairwell, central plaza in front of O-Market, and the backyard, possibly also the areas around Playerment once renovated. The outside areas could also have an exceptions for the regulation to be active only during the day (4:00 - 20:00). This something enforced in Poland and seems fitting to the absurdity of Obenseuer, place full of drunk, homeless people, but police is tasked with patroling and dealing tickets anyway, and people keep drinking anyway. Plus it could add a little of dynamic for players with alcohol addiction, since now they would need to hide behind fences, dumpsters or inside to avoid tickets/crime points.

**Comment by Oskutin** [comment]
Kolhola stairwell is already prohibition zone.

**Comment by Nextej** [comment]
If it is, doesn't seem to be enforced anyhow.

**Comment by Oskutin** [comment]
Needs to be seen on camera

**Comment by Nextej** [comment]
Alright, I've tested it in front of Hostel's camera, but I was just too close.

**Comment by Oskutin** [comment]
Atleast osmo does work

**Comment by Nextej** [comment]
Yeah, Osmo does work, custom brew does not, and also Hostel's camera has a weird range.

**Comment by Nextej** [comment]
Anyway, I'm waiting for it to be added to central plaza, just to slap in the face new players who just brought beer to ease the addiction with "Crime Commited: Public Drinking +50"

**Comment by greensalt99** [comment]
How you Balance around Obenseuer culture? Opensewer is addict district left to its own culture by government.

- Drinking in banned public? how about universal healthcare and place to sleep in warm room, clean water? 
- How about fare prisoner treatment! my prison cell was all dirty and there was dude spying on me through hole!
 oh you cant give up on our demands, because city  overcrowded, but we Obenseuerens get punished and jugged by our drinking culture on fresh air!!!! 
 we sure wont vote for this government next year! 
 #Obenseuer Vote for public free drinking party, vote for government who cares about their citizens! 
 
:)
